README.md

```diff
@@ -18,3 +22,58 @@ Repository for the source code for the Hierarchical Engine for Large-scale Infra
+
+The [HELICS-Tutorial repository](https://github.com/GMLC-TDC/HELICS-Tutorial) provides a series of tutorials using HELICS to build a co-simulation using domain-specific external modeling tools that is built around an electric power system use case with integrated transmission-distribution-market-communication quasi-steady-state-timeseries (QSTS) simulation.
+
+The [HELICS-Use-Cases repository](https://github.com/GMLC-TDC/HELICS-Use-Cases) includes examples for a growing range of research use cases for inspiration.
+
+
+# Documentation
+
+Our GitHub pages provides a rich set of [documentation](https://gmlc-tdc.github.io/HELICS-src/index.html) including a set of introductory [examples](https://gmlc-tdc.github.io/HELICS-src/introduction/index.html), a [developers guide](https://gmlc-tdc.github.io/HELICS-src/developer-guide/index.html), complete doxygen-auto-produced [API documentation](https://gmlc-tdc.github.io/HELICS-src/doxygen/), and more. 
+
+Additionally, our initial requirements document can be found [here](https://github.com/GMLC-TDC/specification-doc/blob/master/src/specification.md), which describes a number of our early design considerations.
+
+# Tools with HELICS support
+
+As a co-simulation framework, HELICS is designed to bring together domain-specific modeling tools so they interact during run time. It effectively tries to build on the sholders of giants by not reinventing trusted simulation tools, but instead, mearly acting as a mediator to coordinate such interactions. HELICS's full power is only apparent when you use it to combine these domain-specific tools.  
+
+Thankfully the HELICS API is designed to be minimally invasive and make it straightforward to connect most any tool that provides either a scripting interface or access to source code. As listed below, a growing set of energy domain tools have HELICS support either natively or through an external interface. We also provide a set of helper apps for various utility and testing purposes.
+
+We are always looking for help adding support for more tools, so please contact us if you have any additions.
+
+#### Power systems Tools
+
+* [GridLAB-D](https://www.gridlabd.org/), distribution power-flow, DER models, basic house thermal and end-use load models, and more. HELICS support currently provided in the [`feature/1024` branch](https://github.com/gridlab-d/gridlab-d/tree/feature/1024) which you have to build yourself as described here
+* [Griddyn](https://github.com/LLNL/GridDyn), a transmission power flow and dynamics simulator. HELICS support currently provided through an experimental branch.
+* [PSST](https://github.com/kdheepak/psst), a python-based unit-commitment and dispatch market simulator. HELICS examples are included in the  [HELICS-Tutorial](https://github.com/GMLC-TDC/HELICS-Tutorial)
+* InterPSS, a Java-based power systems simulator.
```

Comment:
add link to interpss, how far along is the integration with HELICS?

Reply:
Done, also MATPower added. all in 58ceae818a75eaffa6b2d5a36ac5643712e4fc3c